docs/status.md

```diff
@@ -84,7 +84,7 @@ one of the following tracking labels.
 | if                       |      no       |     yes*     |       yes*        |        no         |     no      |
 | imag                     |      no       |     yes*     |       yes*        |        yes        |     no      |
 | infeed                   |      no       |     yes*     |        no         |        no         |     no      |
-| iota                     |      no       |     yes*     |    infeasible     |        yes        |     no      |
+| iota                     |      yes      |     yes      |    infeasible     |        yes        |     yes     |
```

Comment:
Since the new format for ODS description and the checklist were developed after this PR was sent out for review, I forgot one more thing: can you update the ODS description as well?